music21/analysis/neoRiemannian.py

```diff
@@ -758,6 +771,20 @@ def testSnN(self):
         N2 = N(c11)
         self.assertEqual([x.name for x in N2.pitches], ['G#', 'B', 'E'])
 
+    def testInstanciatingChordPCNumbers(self):
+
+        L_cs_maj_symb = L(chord.Chord('C# E# G#'))
+        L_cs_maj_nb = L(chord.Chord([1, 5, 8]))
```

Comment:
`c_sharp_maj_L_named`  and `c_sharp_maj_L_pitch_classes`  -- spell out abbreviations unless they are absolutely clear.  I thought "cs" referred to "chordSymbol" and was very confused.